src/library/vicon_estimator.cpp

```diff
@@ -108,10 +120,20 @@ void TranslationalEstimator::updateEstimate(
   // Saving estimate to intermediate results
   estimator_results_.position_estimate_ = position_estimate_W_;
   estimator_results_.velocity_estimate_ = velocity_estimate_W_;
+    }// If outlier detected just write priori estimate to posteriori
+      else {
```

Comment:
Move comment to next line and de-indent the else statement.

We stick to google coding style. A very simple solution to this and other style problems is to using clang-autoformatter (if you are using sublime). Our instructions on doing this can be found here: https://github.com/ethz-asl/mav_tools/wiki/How-to-Write-a-ROS-Package#chapter-3-coding-standards